.github/workflows/ci.yml

```diff
@@ -0,0 +1,172 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  cmake_test:
+    name: CMake test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: |
+        sudo ./ci/setup_cmake.sh
+        sudo ./ci/setup_ci_environment.sh
+    - name: run tests
+      run: ./ci/do_ci.sh cmake.test
+
+  plugin_test:
+    name: Plugin -> CMake
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: |
+        sudo ./ci/setup_cmake.sh
+        sudo ./ci/setup_ci_environment.sh
+    - name: run tests
+      run: ./ci/do_ci.sh cmake.test_example_plugin
+
+  gcc_48_test:
+    name: Legacy Bazel
+    runs-on: ubuntu-latest
```

Comment:
Let's make this `ubuntu-18.04` instead of `ubuntu-latest`. As soon as `ubuntu-latest` is rolled forward to `ubuntu-20.04`, this would break. There's no gcc 4.8 package anymore on Ubuntu 20.04.

Reply:
Good catch! Fixed.